pages/info/pools/index.tsx

```diff
@@ -0,0 +1,43 @@
+import ICArrowRight from "assets/svg/arrow-right.svg";
+import InfoLayout from "components/Layout/Info";
+import AprByPoolsChart from "components/Pages/Pools/AprByPoolsChart";
+import LiquidityByTokensChart from "components/Pages/Pools/LiquidityByTokensChart";
+import PoolsTable from "components/Pages/Pools/PoolsTable";
```

Comment:
@haoquang24 Path chỗ này mình đặt khác được không e, có `Pages` dễ hiểu nhầm.

Reply:
thế đặt là views | modules nhá a, e thấy source code chúng nó toàn tách thành folder views đồng cấp vs folder components đó a,